site/content/extend/plugins/community_process.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Plugins on Community Server
+date: 2018-10-01T00:00:00-05:00
+subsection: Plugins (Beta)
+weight: 50
+---
+
+
+Getting your plugin onto community.mattermost.com is a valuable source of feedback. However we must ensure that our community server remains stable for everyone. This document outlines the process of getting your plugin onto the community server. In the future, some of these steps will be required to get your plugin into the integrations marketplace (TBD). When you are ready to have your plugin start this process, ask in the ~Toolkit channel on the community server.
```

Comment:
1 - What's a "community server"?
2 - Who owns the process when someone asks in the ~Toolkit channel?
3 - Who owns bug fixes / maintenance when something breaks?
4 - Is this intended for everyone, including contributor-created plugins?